src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/tests/Tabs.test.js

```diff
@@ -21,169 +21,80 @@ Object.defineProperty(window, 'getComputedStyle', {
 
 test('Render a Tabs component with type root', () => {
     const changeSpy = jest.fn();
-
-    expect(render(
+    const {container} = render(
         <Tabs onSelect={changeSpy} selectedIndex={null} type="root">
             <Tabs.Tab>Tab 1</Tabs.Tab>
             <Tabs.Tab>Tab 2</Tabs.Tab>
             <Tabs.Tab>Tab 3</Tabs.Tab>
         </Tabs>
-    )).toMatchSnapshot();
+    );
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render a Tabs component with type nested', () => {
     const changeSpy = jest.fn();
-
-    expect(render(
+    const {container} = render(
         <Tabs onSelect={changeSpy} selectedIndex={null} type="nested">
             <Tabs.Tab>Tab 1</Tabs.Tab>
             <Tabs.Tab>Tab 2</Tabs.Tab>
             <Tabs.Tab>Tab 3</Tabs.Tab>
         </Tabs>
-    )).toMatchSnapshot();
+    );
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render a Tabs component with type inline', () => {
     const changeSpy = jest.fn();
-
-    expect(render(
+    const {container} = render(
         <Tabs onSelect={changeSpy} selectedIndex={null} type="inline">
             <Tabs.Tab>Tab 1</Tabs.Tab>
             <Tabs.Tab>Tab 2</Tabs.Tab>
             <Tabs.Tab>Tab 3</Tabs.Tab>
         </Tabs>
-    )).toMatchSnapshot();
+    );
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render a Tabs component with a selected tab and a badge', () => {
     const changeSpy = jest.fn();
     const selectedTabIndex = 0;
-
-    expect(render(
+    const {container} = render(
         <Tabs onSelect={changeSpy} selectedIndex={selectedTabIndex} type="root">
             <Tabs.Tab>Tab 1</Tabs.Tab>
             <Tabs.Tab>Tab 2</Tabs.Tab>
             <Tabs.Tab badges={[<Badge key="badge1">1</Badge>, <Badge key="badge2">2</Badge>]}>Tab 3</Tabs.Tab>
         </Tabs>
-    )).toMatchSnapshot();
-});
-
-test('Clicking on a Tab should call the onSelect handler', () => {
-    const changeSpy = jest.fn();
-    const selectedTabIndex = 0;
-
-    const tabs = mount(
-        <Tabs onSelect={changeSpy} selectedIndex={null}>
-            <Tabs.Tab>Tab 1</Tabs.Tab>
-            <Tabs.Tab>Tab 2</Tabs.Tab>
-            <Tabs.Tab>Tab 3</Tabs.Tab>
-        </Tabs>
     );
 
-    tabs.find('.tab button').at(0).simulate('click');
-    expect(changeSpy).toHaveBeenCalledWith(selectedTabIndex);
+    expect(container).toMatchSnapshot();
 });
 
-test('Clicking on several non- and collapsed tabs', () => {
```

Comment:
we cannot migrate this test easily as it sets propüerties of the component instance. i think i would propose to remove it for now and investigate how to test this if there is a bug in this part in the code in the future 🙈

Reply:
i tried to implement a simpler version of this test as a replacement. unfortunately, the `Tabs` component uses `offsetWidth` to determine the collapsed tabs but `jsdom` does not support `offsetWidth` at all https://github.com/jsdom/jsdom/issues/135. this means we cannot test if tabs are collapsed because they will never be collapsed in the test environment 😕

Reply:
I think the target of the whole test is to test if at specific window size the dropdown with the rest of tabs is shown. Is that possible in a more simple test or is that already tested in other test case?

Reply:
And it tested if I click one of the dropdown items that then that active element is shown and the other one goes into the dropdown. But If you say it is too much work at current state I'm fine with merging it.

Reply:
the problem is that there will never be a dropdown because the calculations of the components depend on `offsetWidth` and `offsetWidth` is not implemented in jsdom (https://github.com/jsdom/jsdom/issues/135) 😕

Reply:
so testing library is a bad choice for testing something which is not server rendered 🙈

Reply:
at least not if the component depends on size calculations. we decided to keep the test case in a `Tabs.enzyme.test.js` for now.